tests/helper/helper_generic.go

```diff
@@ -81,3 +83,52 @@ func Unindented(jsonStr string) (string, error) {
 	}
 	return string(obj), err
 }
+
+// WatchNonRetCmdStdOut run odo watch and get the cmdSTDOUT output into buffer.
```

Comment:
You can make the comment line more talkative i guess :)

Reply:
Done